farm/modeling/prediction_head.py

```diff
@@ -1654,7 +1654,7 @@ def _embeddings_to_scores(self, query_vectors:torch.Tensor, context_vectors:torc
         softmax_scores = nn.functional.log_softmax(scores, dim=1)
         return softmax_scores
 
-    def logits_to_loss(self, logits: Tuple[torch.Tensor, torch.Tensor], **kwargs):
+    def logits_to_loss(self, logits: dict, **kwargs):
```

Comment:
not sure if we should really switch to a dict here. It's kind of breaking the design with other prediction head / models in FARM

Reply:
reverted back to original